rts/lra/lra-test/pom.xml

```diff
@@ -23,7 +23,12 @@
     <properties>
         <lra.coordinator.exec.plugin.phase>none</lra.coordinator.exec.plugin.phase> <!-- not starting coordinator by default, need -Parq -->
         <lra.coordinator.port>8080</lra.coordinator.port>
-        <lra.coordinator.trace.params></lra.coordinator.trace.params>
+        <lra.coordinator.log.console.disable></lra.coordinator.log.console.disable>
```

Comment:
I'm a bit concerned about so many log properties. Would you mind to merge it under just one? E.g. `lra.coordinator.log.params` or similar? And then put all the configuration on one line rather than splitting it over multiple.

But this could be only my personal feeling on having the properties too bloated.

Reply:
okay, will make the changes

Reply:
I tried merging them, but after that functionality stopped working. So for now I will leave this as it is.